tests/Validator/P0wnedPasswordValidatorTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the RollerworksPasswordStrengthValidator package.
+ *
+ * (c) Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Rollerworks\Component\PasswordStrength\Tests\Validator;
+
+use Rollerworks\Component\PasswordStrength\P0wnedPassword\Request\Client;
+use Rollerworks\Component\PasswordStrength\P0wnedPassword\Request\Result;
+use Rollerworks\Component\PasswordStrength\Validator\Constraints\P0wnedPassword;
+use Rollerworks\Component\PasswordStrength\Validator\Constraints\P0wnedPasswordValidator;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Violation\ConstraintViolationBuilderInterface;
+
+class P0wnedPasswordValidatorTest extends TestCase
+{
+    /** @var Client|MockObject */
+    private $client;
+
+    /** @var P0wnedPassword */
+    private $constraint;
+
+    /** @var ExecutionContextInterface|MockObject */
+    private $context;
+
+    /** @var P0wnedPasswordValidator */
+    private $validator;
+
+    public function setUp()
+    {
+        $this->constraint = new P0wnedPassword();
+        $this->context = $this->createMock(ExecutionContextInterface::class);
+        $this->client = $this->createMock(Client::class);
+        $this->validator = new P0wnedPasswordValidator($this->client);
+        $this->validator->initialize($this->context);
+    }
+
+    public function testFound()
+    {
+        $builder = $this->createMock(ConstraintViolationBuilderInterface::class);
```

Comment:
Please use the `ConstraintValidatorTestCase` provided by Symfony 👍
See https://github.com/rollerworks/PasswordStrengthValidator/blob/master/tests/Validator/PasswordStrengthTest.php for usage (you may also need the `getMock()` method; but I am not sure if this already fixed for older Symfony versions).

This makes testing easier and prevents testing against a mock ViolationsBuilder (which has actually caused some issues in the past).